SimGeneral/TrackingAnalysis/plugins/SimHitTPAssociationProducer.cc

```diff
@@ -44,9 +48,9 @@ void SimHitTPAssociationProducer::produce(edm::Event& iEvent, const edm::EventSe
   }
 
   // PSimHits
-  for (auto psit=_simHitSrc.begin();psit<_simHitSrc.end();++psit) {
+  for (auto const& psit : _simHitSrc ) {
     edm::Handle<edm::PSimHitContainer>  PSimHitCollectionH;
-    iEvent.getByLabel(*psit,  PSimHitCollectionH);
+    iEvent.getByLabel(psit,  PSimHitCollectionH);
```

Comment:
Yes.  However, I am doing the consumes migration centrally for many packages.  Working in this mode at this scale, it is much faster to just do the consumes part so I don't have to be switching gears.  If the powers that be wish, I can do the migration to getByToken after the consumes part is finished.